node/lib/statsd.js

```diff
@@ -291,22 +222,17 @@ TChannelStatsd.prototype.onStat = function onStat(stat) {
 
     var key = getKey(stat);
 
-    switch (stat.type) {
-        case 'counter':
-            return self.statsd.increment(key, stat.value);
-
-        case 'gauge':
-            return self.statsd.gauge(key, stat.value);
-
-        case 'timing':
-            return self.statsd.timing(key, stat.value);
-
-        default:
-            self.channel.logger.error('Trying to emit an invalid stat object', {
-                statType: stat.type,
-                statName: stat.name
-            });
-            break;
+    if (stat.type === 'counter') {
```

Comment:
Because switches are a scourge upon this Earth and Doug Crockford says “no”.

![not found](http://akamaicovers.oreilly.com/images/9780596517748/cat.gif)